plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -3255,7 +3256,7 @@ public boolean configureDefaultNetworkRulesForSystemVm(final Connect conn, final
             return false;
         }
 
-        final Script cmd = new Script(_securityGroupPath, _timeout, s_logger);
+        final Script cmd = new Script(_securityGroupPath, _timeout.getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.